lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1249,19 +1249,17 @@ def _load_nics(
 
     @retry(exceptions=LisaException, tries=150, delay=2)  # type: ignore
     def _load_public_ips(
-        self, environment: Environment, log: Logger
+        self, environment: Environment, log: Logger, resource_group_name: str = ""
```

Comment:
The interface can be refined like below. And the caller should pass in the resource_group_name.
```python
def _load_public_ips(self, resource_group_name: str = "", log: Logger):
```